app/search/template.hbs

```diff
@@ -47,7 +62,48 @@ as |layout|>
         data-analytics-scope='Search page left panel'
         local-class='left-panel'
     >
-        <h2>{{t 'search.left-panel.header'}}</h2>
+        {{#let (unique-id 'two') as |twoId|}}
+            <h2 id={{twoId}}>{{t 'search.left-panel.header'}}</h2>
+            <EmberPopover
+                data-test-search-help-2
+                @targetId={{twoId}}
+                @tooltipClass={{local-class 'search-help-tutorial-2'}}
+                @innerClass={{'search-help-tutorial-inner'}}
```

Comment:
This `@innerClass` can be removed from all of the EmberPopover invocations. It's not applying the local-class that's defined in the scss file here as this is applying a global-class.

Reply:
@futa-ikeda this style is being applied:

![image](https://github.com/CenterForOpenScience/ember-osf-web/assets/82047646/fc696164-a776-434e-99be-3d3c1c3e2fc2)

If you have an alternate working selector you would like me to use, I can do that however this styling is in reference to the global/local tooltip discussion we had previously.